DQMOffline/RecoB/src/MatchJet.cc

```diff
@@ -94,8 +71,18 @@ void MatchJet::matchCollections(
 	recToRef.resize(recJets.size(), -1);
 
 	Matching<double> matching(corrRefJets, corrRecJets,
-	                          JetDistance<Vector, Vector, double>(
-	                          		sigmaDeltaR, sigmaDeltaE));
+                              [this](auto& v1, auto& v2)
+                              {
+                                  using namespace ROOT::Math;
+                          //        return VectorUtil::DeltaR2(v1, v2) / this->sigmaDeltaR2 +
+                          //               sqr(2. * (v1.R() - v2.R()) /
+                          //                        (v1.R() + v2.R())) / this->sigmaDeltaE2;
```

Comment:
This comment seems to be unnecessary as the commented out code is the same as used code apart from the fact that the value is assigned to a variable.